integration-cli/docker_cli_run_test.go

```diff
@@ -2617,7 +2617,7 @@ func TestRunVolumesCleanPaths(t *testing.T) {
 func TestRunSlowStdoutConsumer(t *testing.T) {
 	defer deleteAllContainers()
 
-	c := exec.Command("/bin/bash", "-c", dockerBinary+` run --rm -i busybox /bin/sh -c "dd if=/dev/zero of=/foo bs=1024 count=2000 &>/dev/null; catv /foo"`)
+	c := exec.Command(dockerBinary, "run", "--rm", "-i", "busybox", "/bin/sh", "-c", "dd if=/dev/zero of=/foo bs=1024 count=2000 &>/dev/null; catv /foo")
```

Comment:
Fixed this with `dd if=/dev/zero of=/dev/stdout bs=1024 count=2000 | catv` because at the end test wants to stream just 1024*2*2000 chars.